src/components/ActionsMenu/ActionsMenu.test.tsx

```diff
@@ -209,3 +210,32 @@ test('allows user to clone query', () => {
     ]
   `);
 });
+
+test('allows user to create new query', () => {
+  const {
+    wrapper: { getByText },
+    store,
+  } = render();
+
+  const newQuery = getByText('actions_menu.new_query');
+  fireEvent.click(newQuery);
+
+  expect(store.getActions()).toMatchInlineSnapshot(`
+    Array [
+      Object {
+        "payload": undefined,
+        "type": "@app/CREATE_NEW_QUERY",
+      },
+    ]
+  `);
+});
+
+test("doesn't allow user to create new query inside query browser", () => {
+  const {
+    wrapper: { queryByText },
+  } = render({ isInsideQueryBrowser: true });
+
+  const removeLink = queryByText('actions_menu.new_query');
```

Comment:
Minor thing - variable could be renamed as current name suggests that we are looking for `removing` query link instead of creating new query.

Reply:
Thx, fixed